### PR TITLE
Update contributing guide

### PR DESCRIPTION
**Pull request by adpi2**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,172 +1,120 @@
...
 
-### Download ivy.xml of SBT plugins from Bintray
-
-This step also requires a premium Bintray.
+The server will start fetching information from Github progressively, you will see them appear after some time.
 
-```
-data/reStart sbt
-```
+After you stop and start the server again, the github information will still be there.
```
Comment (mlachkar) [inline CONTRIBUTING.md:73]: because there are store in this directory locally ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,172 +1,120 @@
...
-This step downloads GitHub metadata and content.
+In the Scaladex terminology, an artifact is the POM file of a Scala artifact and a project is the Github repository of a Scala project.
+An artifact must be associated to a single project.
+A project can have many artifacts.
 
-You need to create a token on your github account: https://github.com/settings/tokens/new
+Locally, we do not receive any POM files, instead we use the `data / run init` to populate the database with some initial artifacts and projects.
 
-Then fill the following file `../scaladex-dev-credentials/application.conf` with:
+While running, the server executes some scheduled jobs periodically to update the information of the projects and to synchronize the search engine:
```
Comment (mlachkar) [inline CONTRIBUTING.md:103]: It makes think that locally, developper wont' be admin.. maybe we should add isAdmin = env.isLocal or part of scala center

**Review by mlachkar — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (adpi2) [reply CONTRIBUTING.md:103]: Oh yes that's a good idea

File: CONTRIBUTING.md (hunk 1 above)
Comment (adpi2) [reply CONTRIBUTING.md:73]: :+1:

**Pull request merged**